.ci/verify

```diff
@@ -13,10 +13,6 @@ cd "$(dirname $0)/.."
 git config --global user.email "gardener@sap.com"
 git config --global user.name "Gardener CI/CD"
 
-# TODO: use pre-built image instead of installing in every pipeline run
-apt-get update
-apt-get install -y unzip jq parallel
```

Comment:
I assume `.ci/verify` is still used by concourse for the release jobs. Will this change work there?

Reply:
yes, I updated branch.cfg ([ref](https://github.com/gardener/gardener/blob/d87fbf58414caec8005f0bd2ae911e7c91957768/branch.cfg#L71), [ref](https://github.com/gardener/gardener/blob/d87fbf58414caec8005f0bd2ae911e7c91957768/branch.cfg#L140)) accordingly